sloggers/slogtest/assert/assert.go

```diff
@@ -37,9 +40,39 @@ func Success(t testing.TB, err error, name string) {
 }
 
 // True asserts act == true.
-//
-// If act isn't true, it will fatal the test.
 func True(t testing.TB, act bool, name string) {
 	slog.Helper()
 	Equal(t, true, act, name)
 }
+
+// Error asserts err != nil.
+func Error(t testing.TB, err error, name string) {
+	slog.Helper()
+	if err == nil {
+		slogtest.Fatal(t, "expected error",
+			slog.F("name", name),
+		)
+	}
+}
+
+// ErrorContains asserts err != nil and err.Error() contains sub.
+//
+// The match will be case insensitive.
+func ErrorContains(t testing.TB, err error, sub, name string) {
```

Comment:
I think I would prefer a function to ensure the two unwrapped errors are equal instead of just comparing strings

I often have to do things like this
https://github.com/cdr/enterprise/blob/master/manager/internal/database/teams_test.go#L46

Additionally, if you call `xerrors.Unwrap` on a non-wrapped error it returns nil instead of the input err. Would be nice to handle that here.

Reply:
I was using ErrorContains in nhooyr/websocket but I can easily fix that. Will adjust this function.